.github/workflows/ur-build-hw.yml

```diff
@@ -126,7 +126,7 @@ jobs:
 
     - name: Download DPC++
```

Comment:
my thought here is - when we first introduced this workflow it was the easiest way of using dpc++ in UR workflow. We didn't want to introduce too much changes in CI... but now, almost one year later we're still using this approach. It doesn't look too good, am I wrong?

Perhaps, we should use nightly docker images or integrate this workflow into `sycl-linux-precommit` and make use of `toolchain_artifact`...?

Reply:
ok, thx. Just FYI, if ever need to change this, I believe switching to nightly images would be the simplest solution.